### PR TITLE
ssh-audit - move off of python2

### PR DESCRIPTION
**Pull request by dtrodrigues**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by fxcoudert** [comment]
Can this possibly work with system Python? (i.e. remove the `python` dependency altogether)

**Comment by mjfwalsh** [comment]
I did a Vermin check a while back and it said ssh-audit will work Python 2.4+ which has been around since Leopard, so it's fairly safe to remove. Unless you want to go with:

    depends_on "python" if MacOS.version <= :tiger

File: Formula/ssh-audit.rb [hunk 1]
```diff
@@ -8,7 +8,7 @@ class SshAudit < Formula
 
   bottle :unneeded
 
-  depends_on "python@2"
+  depends_on "python"
```
Comment (zbeekman) [inline Formula/ssh-audit.rb]:
```suggestion
```

**Review by zbeekman — CHANGES_REQUESTED** [review]

File: Formula/ssh-audit.rb (hunk 1 above)
Comment (zbeekman) [reply Formula/ssh-audit.rb]: should work with system python. Unless you *really* want to be using it with py3...

**Comment by dtrodrigues** [comment]
I removed the python dependency per suggestion of @zbeekman

File: Formula/ssh-audit.rb [hunk 2]
```diff
@@ -8,7 +8,6 @@ class SshAudit < Formula
 
   bottle :unneeded
```
Comment (zbeekman) [inline Formula/ssh-audit.rb]: Sorry, need to fix the audit failure too. You can try this, but `brew audit --strict --fix ssh-audit` might be a safer bet.

```suggestion
```

**Review by zbeekman — CHANGES_REQUESTED** [review]
Fix audit failures please

**Comment by dtrodrigues** [comment]
It passes `brew --audit --strict` now.

**Comment by zbeekman** [comment]
Thanks for your contribution to Homebrew! 🎉 🥇 

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request closed without merging**